server/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
+
+module.exports = {
+    mode: 'production',
+    entry: './dist/index.js',
+    target: "node",
+    output: {
+        path: path.resolve(__dirname, 'bundle'),
+        filename: 'server.bundle.js'
```

Comment:
@BHRnR our whole application will be bundled into a single js file, which can be run with `node server.bundle.js` without the need to install the npm dependenciese (all our dependencies and usaged are bundled into this file)